### DISCUSSION TITLE
Combine key and a command in run

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1062] [Q&A] [answered]
How to do scrollToTop and then run another command? I could only find if; then or if; else combinations. How to run 2 commands back to back?

EDIT: I'm using `run xxx gg?<cmd>:<cmd>`. \<cmd\> is a command with options. Is there a better way to do this?

**Comment by gdh1995** [comment, marked as answer]
If you mean to "run a second command whatever result a first command returns (whether it succeeds or not)`, then you may:
```
# wrap a first command with `(` and `)`
run xxx (gg)cmd#arg1=value1

# or just use `?` and `:`
map yyy cmd arg1=value1
run xxx gg?yyy:yyy
```

**Reply by limaceous-bushwhacker** [reply]
Thank you. I was actually using the second one but didn't think of mapping the command on its own.
